util/src/main/java/com/google/cloud/hadoop/util/EntriesCredentialConfiguration.java

```diff
@@ -57,6 +61,13 @@
    * service accounts. The default is to use a service account.
    */
   public static final String ENABLE_SERVICE_ACCOUNTS_SUFFIX = ".auth.service.account.enable";
+
+  // TODO Document the parameters. Additional documentation in gs-defaults.xml, toString
+  public static final String SA_PRIVATE_KEY_ID_SUFFIX = ".auth.service.account.private.key.id";
```

Comment:
Change to `SERVICE_ACCOUNT_...` for consistency

Reply:
Done.